mod/notifications/start.php

```diff
@@ -41,6 +41,10 @@ function notifications_plugin_init() {
 function notifications_page_handler($page) {
 
 	elgg_gatekeeper();
+
+	// Set the context to settings
+	elgg_set_context('settings');
+
```

Comment:
If it were easier to add edges to existing routes, we would simply not use a custom handler for notifications, and just follow the route pattern of the settings page.